storybookjs/storybook
# Add built storybooks to official storybook

Issue:

Screenshot tests for example apps were difficult to work with.

## What I did

Added a iframe story to the official storybook for each example app. This results in these tests getting picked up by Chromatic.

## How to test

Check these screenshots are correct: https://www.chromaticqa.com/build?appId=5a375b97f4b14f0020b0cda3&number=984 -- actually I am not sure what is going on with polymer there, perhaps I shouldn't have added it too?

I'm not completely on top of what the previous tests were used for (and I haven't actually removed them in this PR, maybe I should), but I wanted to try this out and run it by you to see if you think it's useful.
